runtime-decompiler/src/main/java/org/jrd/backend/data/Cli.java

```diff
@@ -262,13 +268,28 @@ private void overwrite() throws Exception {
         }
     }
 
+    @SuppressFBWarnings(value = "OS_OPEN_STREAM", justification = "The stream is clsoed as conditionally as is created")
```

Comment:
Probably could be *cheesed* by having the condition as a `final boolean` variable. Probably.